.github/workflows/unix.yml

```diff
@@ -132,10 +134,18 @@ jobs:
         run: |
           ${system}_build_plugin_pkg ${{ matrix.cubit }}
 
-      - name: Upload package
+      - if: github.event_name != 'release'
+        name: Upload package
```

Comment:
where does this get uploaded?

Reply:
should have been upload there: https://github.com/bam241/Trelis-plugin/releases/tag/0.0.8

but there is a problem....

Reply:
basically we should end up with something like:
https://github.com/bam241/Trelis-plugin/releases/tag/0.0.7

Reply:
Wait I'm confused too, this block gets executed when the event is not a release right? Do we want to upload anything in that case?

Reply:
this is upload the artifact at the job summary location they're are useful for debug when running CI, 
on release we use the other upload mechanism

Reply:
Ahhh I see. Very nice! Didn't know that was a thing.